README.md

```diff
@@ -63,6 +67,16 @@ To run the tests and see the results in your terminal, run:
 bundle exec rake jasmine:ci
 ```
 
+## Issue with FFI on OSX Mojave
+
+Users on OSX Mojave (10.14) may get this error when running `bundle exec middleman serve` on apps that use this gem.
+
+There are 3 different ways to solve this, depending on what is possible for you. Listed here as best to worst:
+
+1. upgrade to macOS 10.15 (Catalina) or higher
+2. tell rubygems not to use the system ffi (gem inst ffi -- --disable-system-libffi)
+3. pin the ffi version back to 1.12.2 in the Gemfile of your app
+
```

Comment:
Recommend we make the numbered bullets in line with style, and be more explicit in our instruction. For example:

"There are 3 possible ways to solve this. The following list is ranked from best to worst.

1. Upgrade to macOS 10.15 (Catalina) or higher. 
2. Tell rubygems not to use the system ffi by running `gem inst ffi -- --disable-system-libffi`
3. Pin the ffi version back to 1.12.2 by editing the Gemfile of your app.

I'm not sure how correct this is, for example, when and where do you run the `gem inst` command? But I would recommend being more explicit.

Reply:
I think these work as you wrote them. One tiny point: don't list items need to be lowercase? I'm going by [the GOVUK styleguide on bullets points and steps](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps).

Reply:
Sorry, to answer your question, you'd run that command after running `bundle exec middleman serve` and it throwing the error mentioned. It would re-install the gem but with the `--disable-system-libffi` flag set to configure the installation process.

Reply:
Looks like `gem inst` is a secret shorthand alias for gem install 😅 

```
% ▶ gem inst -h | head -n 1
Usage: gem install GEMNAME [GEMNAME ...] [options] -- --build-flags [options]
```

The long form (gem install) might be less confusing.

Reply:
`gem i` and `gem ins` also work, but `gem in` is apparently ambiguous (between `gem info` and `gem install`)

Reply:
Ah, I was thinking of numbered steps rather than bullets. I guess these aren't steps in a process, they are options. Maybe we could make them unordered bullets e.g.

"There are 3 possible ways to solve this. From best to worst, you can:

- upgrade to macOS 10.15 (Catalina) or higher
- tell rubygems not to use the system ffi by running `gem inst ffi -- --disable-system-libffi` in the command line when the error shows
- pin the ffi version back to 1.12.2 by editing the Gemfile of your app"

Reply:
@jonathanglassman I've plugged in those changes and changed `gem inst` to `gem install` (thanks for the explanation @richardTowers) so marking this as resolved but feel free to undo that if anything outstanding/wrong.